File: scratch.js
require('dotenv').config();
const { createClient } = require('@supabase/supabase-js');
const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_SERVICE_KEY);
async function run() {
  const { data, error } = await supabase.from('products').select('sku, url_key, product_url').eq('sku', '55562599-280');
  console.log(data, error);
}
run();

File: scratch.py
import os
import asyncio
from supabase import create_client, Client
from dotenv import load_dotenv

load_dotenv()
url = os.environ.get("SUPABASE_URL")
key = os.environ.get("SUPABASE_SERVICE_KEY")
supabase: Client = create_client(url, key)

response = supabase.table("products").select("sku, url_key, product_url").eq("sku", "55562599-280").execute()
print(response.data)

response2 = supabase.table("products").select("sku, url_key, product_url").eq("url_key", "55562599-280").execute()
print("By url_key:", response2.data)
